Makefile

```diff
@@ -8,6 +8,7 @@ ANSIBLE_PYTHON ?= /usr/bin/python3
 AP ?= ansible-playbook -vv -c local -i localhost, -e ansible_python_interpreter=$(ANSIBLE_PYTHON)
 PATH_TO_SECRETS ?= $(CURDIR)/secrets/
 COV_REPORT ?= term-missing
+COLOR ?= yes
```

Comment:
What is the problem with colors?

Reply:
As mentioned in the commit message, color codes.

This is a log from Zuul before this change:

```
2020-06-03 00:17:59.545122 | test-node | PYTHONPATH=/src-packit-service PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --color=yes --verbose --showlocals --cov=packit_service --cov-report=term-missing ./tests/unit ./tests/integration/
2020-06-03 00:18:02.234837 | test-node | [1m============================= test session starts ==============================[0m
2020-06-03 00:18:02.234994 | test-node | platform linux -- Python 3.7.7, pytest-5.4.3, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python3
2020-06-03 00:18:02.242672 | test-node | cachedir: .pytest_cache
2020-06-03 00:18:02.242765 | test-node | rootdir: /src-packit-service, inifile: pytest.ini
2020-06-03 00:18:02.242779 | test-node | plugins: cov-2.9.0, flask-1.0.0, celery-4.4.3
2020-06-03 00:18:02.848340 | test-node | [1mcollecting ... [0mcollected 215 items
2020-06-03 00:18:02.849016 | test-node |
2020-06-03 00:18:02.851955 | test-node | tests/unit/test_api.py::test_optional_time[input_object0-str] [32mPASSED[0m[33m     [  0%][0m
2020-06-03 00:18:02.854230 | test-node | tests/unit/test_api.py::test_optional_time[None-NoneType] [32mPASSED[0m[33m         [  0%][0m
2020-06-03 00:18:02.856817 | test-node | tests/unit/test_build_helper.py::test_targets[build_with_targets] [32mPASSED[0m[33m [  1%][0m
```
and after

```
2020-06-04 06:17:59.136390 | test-node | PYTHONPATH=/src-packit-service PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --color=no --verbose --showlocals --cov=packit_service --cov-report=term-missing ./tests/unit ./tests/integration/
2020-06-04 06:18:02.239529 | test-node | ============================= test session starts ==============================
2020-06-04 06:18:02.239630 | test-node | platform linux -- Python 3.7.7, pytest-5.4.3, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python3
2020-06-04 06:18:02.247235 | test-node | cachedir: .pytest_cache
2020-06-04 06:18:02.247270 | test-node | rootdir: /src-packit-service, inifile: pytest.ini
2020-06-04 06:18:02.247281 | test-node | plugins: flask-1.0.0, celery-4.4.4, cov-2.9.0
2020-06-04 06:18:02.942658 | test-node | collecting ... collected 213 items
2020-06-04 06:18:02.943638 | test-node |
2020-06-04 06:18:02.946948 | test-node | tests/unit/test_api.py::test_optional_time[input_object0-str] PASSED     [  0%]
2020-06-04 06:18:02.949354 | test-node | tests/unit/test_api.py::test_optional_time[None-NoneType] PASSED         [  0%]
2020-06-04 06:18:02.952408 | test-node | tests/unit/test_build_helper.py::test_targets[build_with_targets] PASSED [  1%]
```

Reply:
But it looks like the `Console` tab does not have a problem with that. Only the `raw` log. But it still makes sense to not use colours.

Reply:
I rarely watch the console tab. Is that the main way people interact with Zuul logs? Should I rather remove this?

Reply:
I can't speak of others, but my point:
- It's more structure and does not contain the beginnings of the line (e.g. `2020-06-04 06:18:02.952408 | test-node | `.
- It does not load the whole log sometimes and I need to switch to the `raw` one => I would still remove the colours.
- I can't find any example where colours are actually displayed => it looks like they read the colour symbols properly, but don't visualize them.